docs/guide/inputs/README.md

```diff
@@ -188,3 +188,53 @@ Event               | Description
 `@input`            | Emitted every time the value of the field changes. Typically this is used implicitly by using `v-model`.
 `@validation`       | Emitted anytime the state of validation changes for an input, irregardless of the visibility of the errors (`v2.3+`). [More details](/guide/validation/#validation-event)
 `@error-visibility` | Emitted when the visibility of the errors changes, for example, on blur or submit (`v2.3+`).
+
+## Context object
+
+All inputs contain a comprehensive context object that details nearly everything
+about the operation of the input. It is passed into all [custom inputs](/guide/inputs/custom-inputs/)
+and slots.
+
+Property        | Description
+----------------|---------------------------------------------------------------
+`addLabel`      | The label to display inside "add more" button on `group` inputs.
+`attributes`    | An object of non-prop attributes passed to the input like `placeholder`
+`blurHandler`   | Function that must be called when an input blurs.
+`classification`| The classification of the input.
+`disableErrors` | `Boolean` indicating if errors should be _displayed_ for this field (defaults to `false`).
+`errors`        | Errors set via the form [error handling](/guide/forms/error-handling/) or directly on the input itself via `error` or `errors` props. Does not include validation errors.
+`hasValue`      | `Boolean` indicates if the field has a valid value.
+`hasLabel`      | `Boolean` indicating if the field has a label prop, `button` classification is always `false`.
+`hasValidationErrors` | Function, returns a `Promise` that resolves to a `Boolean`.
+`help`          | The `help` prop value (help text)
+`helpPosition`  | The position of the help text, `before` or `after` the element wrapper. Defaults to `before`.
+`getValidationErrors` | Function, returns a `Promise` that resolves to an array of validation error objects.
+`id`            | The id prop or an auto-generated id.
+`imageBehavior` | The value of the `image-behavior` prop. Defaults to `preview`.
+`isSubField`    | A function that returns a `boolean` indicating if it is a descendant of a `group` type.
+`label`         | The value of the `label` prop.
+`labelPosition` | The position of the label, `before` or `after`. Default is `before` for all except `box` classified inputs. Can be overridden with `label-position` prop.
+`limit`         | For a `group` type, this is the is the maximum number of repeatable items allowed (default is `Infinity`).
+`model`         | The value of the current field, bound to a setter.
+`name`          | The name of the field, if none is set, it auto-generates a name.
+`options`       | The `options` prop converted to an array (when applicable).
```

Comment:
Let's change "is executed on every input even already" to "is executed on every input event automatically"